scripts/onboarding/aks/multi-tenancy-multiclusters/README.md

```diff
@@ -0,0 +1,85 @@
+# Multi-Tenant Container Insights Setup with Shared DCR/DCE
+
+This guide explains how to set up Container Insights monitoring for multiple AKS clusters using a shared Data Collection Rule (DCR) and Data Collection Endpoint (DCE).
+
+## Setup Process
+
+### 1. Deploy Shared DCR and DCE
+
+First, deploy the shared DCR and DCE using the `existingClusterOnboarding.json` template. This creates the core monitoring infrastructure that will be shared across multiple clusters.
+
+```bash
+az deployment group create \
+  --name shared-monitoring-setup \
```

Comment:
configDCE has to be same region/location as AKS cluster hence you would need to take out the config DCE from this shared template and ask the customer to create one to one with AKS cluster.